Marlin/Configuration.h

```diff
@@ -413,6 +413,13 @@
 #define TEMP_SENSOR_5 0
 #define TEMP_SENSOR_BED 0
 #define TEMP_SENSOR_CHAMBER 0
+#define TEMP_SENSOR_PROBE 0
+/* Don't forget to set a pin for your probe thermistor
+ * In my pins_BTT_SKR.h I have added:
+ * #if HOTENDS == 1 && TEMP_SENSOR_PROBE != 0
+ *   #define TEMP_PROBE_PIN TEMP_1_PIN
+ * #endif
+ */
 
```

Comment:
Do you have a suggestion for what existing sensor type number is similar to PINDAs temperature sensor, to get a realistic temperature.

Reply:
Prusa's MK3 firmware uses `TEMP_SENSOR_PINDA 1` (100k thermistor - best choice for EPCOS 100k (4.7k pullup))

Reply:
I currently use the one thisiskeithb suggested (as I copied it from Prusa firmware) and it works fine so far (readings look plausible).